### ISSUE TITLE
@grpc/grpc-js Support

### ISSUE THREAD

**Issue by moloch--** [improbable-eng/ts-protoc-gen#226]
### Feature Request

Support generation of code that works with `@grpc/grpc-js` instead of `grpc-node` as the developers have indicated they plan to deprecate `grpc-node`: https://grpc.io/blog/grpc-js-1.0/

Simply replacing the imports does not seem to work with the newly generated code/library.

**Comment by 117sparta** [comment]
I guess you can modify the code and meet your need.

**Comment by 117sparta** [comment]
![image](https://user-images.githubusercontent.com/37238800/83242908-baa68980-a1cf-11ea-876e-cc4df0ef7430.png)
The only thing you need to do is to modify one line of code in grpcnode.ts file.

**Comment by 117sparta** [comment]
Find the code below and change "import * as grpc from 'grpc" to "import * as grpc from '@grpc/grpc-js'"
```
// Import statements.
  serviceDescriptor.imports
    .forEach(importDescriptor => {
      printer.printLn(`import * as ${importDescriptor.namespace} from "${importDescriptor.path}";`);
    });
  printer.printLn(`import * as grpc from "grpc";`);
```
after that, run npm run install and npm run  build on terminal.
Then include the bin folder and lib folder which is generated after run the "npm run build" command into your project. Modify the executable path of the shell command below to the bin folder.

```
# Path to this plugin, Note this must be an abolsute path on Windows (see #15)
PROTOC_GEN_TS_PATH="the bin folder path"

# Path to the grpc_node_plugin
PROTOC_GEN_GRPC_PATH="./node_modules/.bin/grpc_tools_node_protoc_plugin"

# Directory to write generated code to (.js and .d.ts files)
OUT_DIR="./generated"

protoc \
    --plugin="protoc-gen-ts=${PROTOC_GEN_TS_PATH}" \
    --plugin=protoc-gen-grpc=${PROTOC_GEN_GRPC_PATH} \
    --js_out="import_style=commonjs,binary:${OUT_DIR}" \
    --ts_out="service=grpc-node:${OUT_DIR}" \
    --grpc_out="${OUT_DIR}" \
    users.proto base.proto
```

Oh, don't forget to install 'google-protobuf' if you are not.

**Comment by zetaron** [comment]
Reference #234

**Comment by badsyntax** [comment]
I've [submitted a PR](https://github.com/improbable-eng/ts-protoc-gen/pull/236) that introduces a `mode` param allowing you to opt-in to using `@grpc/grpc-js` instead of `grpc`, for example:

```
--ts_out=service=grpc-node,mode=grpc-js
```

**Comment by MarcusLongmuir** [comment]
Fixed in #236
